### ISSUE TITLE
undo behaviour

### ISSUE THREAD

**Issue by adamjimenez** [convergencelabs/ace-collab-ext#26]
I've been playing with the Ace demo. I notice when collaborating if you press undo it undoes the other user's changes, which would be confusing when collaborating.

I also observed the following.
user 1: enter 'a', press undo. 'a' disappears as you would expect.
user 2: press undo. 'a' reappears.

**Comment by alalonde** [comment]
Yes, this is called "local undo" and can be a very difficult problem to solve in a distributed-state environment. It is on our [roadmap](https://github.com/convergencelabs/convergence-project/wiki/Convergence-Road-Map).

**Comment by adamjimenez** [comment]
I was originally working with Firepad which is no longer supported so I've been looking around for an alternative. I know it's not easy and I will follow your progress with great interest.

**Comment by mmacfadden** [comment]
The undo behavior will largely be added into Convergence itself, and then we will update this library if it needs changes to support that.
